diff --git a/pip_pkg.sh b/pip_pkg.sh
index 6a8671b..cf81fc7 100755
--- a/pip_pkg.sh
+++ b/pip_pkg.sh
@@ -40,7 +40,7 @@ echo >>tensorflow_probability/python/version.py \
 # specifies the output dir) to setup.py, e.g.,
 #  ./pip_pkg /tmp/tensorflow_probability_pkg --gpu --release
 # passes `--gpu --release` to setup.py.
-python setup.py bdist_wheel --universal ${@:2} --dist-dir="$DEST" >/dev/null
+python3 setup.py bdist_wheel --universal ${@:2} --dist-dir="$DEST" --release >/dev/null
 
 set +x
 echo -e "\nBuild complete. Wheel files are in $DEST"
