diff --git a/include/crt/host_config.h b/include/crt/host_config.h
index e591153..bd1c5d9 100644
--- a/include/crt/host_config.h
+++ b/include/crt/host_config.h
@@ -124,11 +124,11 @@
 
 #if defined(__GNUC__)
 
-#if __GNUC__ > 7
+#if __GNUC__ > 8
 
-#error -- unsupported GNU version! gcc versions later than 7 are not supported!
+#error -- unsupported GNU version! gcc versions later than 8 are not supported!
 
-#endif /* __GNUC__ > 7 */
+#endif /* __GNUC__ > 8 */
 
 #if defined(__APPLE__) && defined(__MACH__) && !defined(__clang__)
 #error -- clang and clang++ are the only supported host compilers on Mac OS X!
