diff --git a/cuda-toolkit/targets/x86_64-linux/include/crt/host_config.h b/cuda-toolkit/targets/x86_64-linux/include/crt/host_config.h
index 27c4e8b..0b9d319 100644
--- a/cuda-toolkit/targets/x86_64-linux/include/crt/host_config.h
+++ b/cuda-toolkit/targets/x86_64-linux/include/crt/host_config.h
@@ -133,11 +133,11 @@
 
 #if defined(__GNUC__)
 
-#if __GNUC__ > 8
+#if __GNUC__ > 9
 
-#error -- unsupported GNU version! gcc versions later than 8 are not supported!
+#error -- unsupported GNU version! gcc versions later than 9 are not supported!
 
-#endif /* __GNUC__ > 8 */
+#endif /* __GNUC__ > 9 */
 
 
 #if defined(__clang__) && !defined(__ibmxl_vrm__) && !defined(__ICC) && !defined(__HORIZON__) && !defined(__APPLE__)
