diff --git a/tools/build_pytorch_libs.py b/tools/build_pytorch_libs.py
index 432fed3bc..33cb1c79b 100644
--- a/tools/build_pytorch_libs.py
+++ b/tools/build_pytorch_libs.py
@@ -190,7 +190,8 @@ def run_cmake(version,
         USE_QNNPACK=USE_QNNPACK,
         USE_TENSORRT=check_env_flag('USE_TENSORRT'),
         USE_FFMPEG=check_env_flag('USE_FFMPEG'),
-        USE_SYSTEM_EIGEN_INSTALL="OFF",
+        USE_SYSTEM_EIGEN_INSTALL="ON",
+        BUILD_CUSTOM_PROTOBUF="OFF",
         USE_MKLDNN=USE_MKLDNN,
         USE_NCCL=USE_NCCL,
         NCCL_EXTERNAL=USE_NCCL,
@@ -206,6 +207,7 @@ def run_cmake(version,
         USE_REDIS=os.getenv('USE_REDIS'),
         USE_GLOG=os.getenv('USE_GLOG'),
         USE_GFLAGS=os.getenv('USE_GFLAGS'),
+        USE_ZMQ=check_env_flag('USE_ZMQ'),
         WERROR=os.getenv('WERROR'))
 
     if USE_GLOO_IBVERBS:
