diff --git a/caffe2/CMakeLists.txt b/caffe2/CMakeLists.txt
index 93f8917..31f3b61 100644
--- a/caffe2/CMakeLists.txt
+++ b/caffe2/CMakeLists.txt
@@ -330,6 +330,10 @@ if(USE_CUDA)
       caffe2_gpu PRIVATE ${Caffe2_GPU_INCLUDE})
   target_link_libraries(
       caffe2_gpu PRIVATE ${Caffe2_CUDA_DEPENDENCY_LIBS})
+  if(USE_FFMPEG)
+      target_link_libraries(
+          caffe2_gpu PRIVATE ${FFMPEG_LIBRARIES} ${OpenCV_LIBS})
+  endif()
 
   # These public dependencies must go after the previous dependencies, as the
   # order of the libraries in the linker call matters here when statically
diff --git a/caffe2/video/CMakeLists.txt b/caffe2/video/CMakeLists.txt
index 204f816..1840f98 100644
--- a/caffe2/video/CMakeLists.txt
+++ b/caffe2/video/CMakeLists.txt
@@ -2,6 +2,9 @@ if(USE_OPENCV AND OpenCV_FOUND AND USE_FFMPEG AND FFMPEG_FOUND)
         message(STATUS "Including video processing operators")
   # ---[ GPU files
   # ------[ general GPU
+  file(GLOB tmp optical_flow.cc video_decoder.cc video_io.cc)
+  set(Caffe2_GPU_SRCS ${Caffe2_GPU_SRCS} ${tmp})
+  #set(Caffe2_GPU_SRCS ${Caffe2_GPU_SRCS} video_io.cc)
   file(GLOB tmp *_gpu.cc)
   set(Caffe2_GPU_SRCS ${Caffe2_GPU_SRCS} ${tmp})
   # ------[ CUDA sources
diff --git a/torch/lib/c10d/test/CMakeLists.txt b/torch/lib/c10d/test/CMakeLists.txt
index ea582ce..04d258c 100644
--- a/torch/lib/c10d/test/CMakeLists.txt
+++ b/torch/lib/c10d/test/CMakeLists.txt
@@ -26,7 +26,7 @@ else()
   c10d_add_test(ProcessGroupGlooTest.cpp c10d c10d)
 endif()
 
-if(MPI_FOUND)
-  add_definitions(-DMPIEXEC=${MPIEXEC})
-  c10d_add_test(ProcessGroupMPITest.cpp c10d)
-endif()
+#if(MPI_FOUND)
+#  add_definitions(-DMPIEXEC=${MPIEXEC})
+#  c10d_add_test(ProcessGroupMPITest.cpp c10d)
+#endif()
