diff --git a/third_party/onnx-tensorrt/CMakeLists.txt b/third_party/onnx-tensorrt/CMakeLists.txt
index e6db9b7..9d31df1 100644
--- a/third_party/onnx-tensorrt/CMakeLists.txt
+++ b/third_party/onnx-tensorrt/CMakeLists.txt
@@ -20,7 +20,7 @@
 
 cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
 # The version of CMake which is not compatible with the old CUDA CMake commands.
-set(CMAKE_VERSION_THRESHOLD "3.10.0")
+set(CMAKE_VERSION_THRESHOLD "3.20.0")
 
 if(${CMAKE_VERSION} VERSION_LESS ${CMAKE_VERSION_THRESHOLD})
   project(onnx2trt LANGUAGES CXX C)
